src/safeds_runner/server/main.py

```diff
@@ -163,18 +167,20 @@ def ws_main(ws: simple_websocket.Server, pipeline_manager: PipelineManager) -> N
                     logging.warning("Invalid message type: %s", received_object.type)
 
 
-def send_websocket_message(connection: simple_websocket.Server, message: Message) -> None:
+def send_websocket_message(connections: list[simple_websocket.Server], message: Message) -> None:
```

Comment:
```suggestion
def broadcast_message(connections: list[simple_websocket.Server], message: Message) -> None:
```

Better conveys that the message is sent to all clients.